specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/examples/unregisterFeature.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "resourceProviderNamespace": "Resource Provider Namespace",
+    "featureName": "feature",
+    "api-version": "2015-12-01",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "name": "Feature1",
+        "properties": {
+          "state": "unregistered"
+        },
+        "id": "feature_id1",
+        "type": "type1"
+      }
+    }
+  }
```

Comment:
Please could you update the example with data that looks like real-life data (e.g. a dummy guid for the "subscriptionid", dummy resourceid for the "id") etc?